examples/persistent-storage/p6/.gn

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2021 Project CHIP Authors
```

Comment:
Persistent-storage  example app does not seem to match PR description (support network commissionig).

Could we split the PRs?

Reply:
Feels like we can follow up on this later?